ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -1,22 +1,48 @@
 import _ from 'lodash';
 import { Command } from '../Command';
 import { type EvalCommandReturn } from './EvalCommand';
+import { NAME, ConfigError } from '../utils';
 
 export type ExportCommandReturn = {
   [key: string]: string;
 };
 
 export type ExportCommandParameters = {
   pipe: EvalCommandReturn;
+  keys?: (key: string) => string;
 };
 
 export class ExportCommand extends Command<ExportCommandReturn> {
   constructor(public parameters: ExportCommandParameters) {
     super(parameters);
   }
 
+  private mutateKeys(result: EvalCommandReturn): EvalCommandReturn {
+    const { keys } = this.parameters;
+
+    if (!keys) {
+      return result;
+    }
+
+    return _.mapKeys(result, (current, key) => {
+      const errorScope: [string, string][] = [
+        ['ExportCommand', ''],
+        ['ConfigKey', key],
+      ];
+      try {
+        const mutatedKey = keys(key);
+        if (!NAME(mutatedKey)) {
+          throw new Error(`key "${mutatedKey}" mustn't contain reserved words`);
+        }
+        return mutatedKey;
+      } catch (error) {
+        throw new ConfigError('invalid config key', error.message, errorScope);
+      }
+    });
+  }
+
   async run() {
     const { pipe } = this.parameters;
-    return _.mapValues(pipe, (current) => current.result.value);
+    return _.mapValues(this.mutateKeys(pipe), (current) => current.result.value);
```

Comment:
For better debugging experience

```suggestion
    const configDict = _.mapValues(pipe, (current) => current.result.value);
    const keyMutatedConfigDict = this.mutateKeys(configDict);
    return keyMutatedConfigDict;
```